docs/source/reference/yaml-spec.rst

```diff
@@ -331,3 +331,28 @@ Available fields:
 
       # Demoing env var usage.
       echo Env var MODEL_SIZE has value: ${MODEL_SIZE}
+
+Experimental
```

Comment:
Should we also add a tip box to config.rst linking to this page? Something like "You can also override some config fields on a per-task basis through the [experimental `config_overrides` field](link)"

Reply:
Good idea. Added.